_pages/community.html

```diff
@@ -90,6 +90,11 @@ <h5 class="mb-1">@CompSoc</h5>
                         <img src="https://pbs.twimg.com/profile_images/843977653732495360/RLdA8AiG_400x400.jpg" class="img img-fluid" style="border-radius: 50%; width: 25%; margin-right: 10px; max-width: 100px"></img>
                         <h5 class="mb-1">@hacktheburgh</h5>
                     </a>
+                    <!-- SIGINT -->
+                    <a class="list-group-item list-group-item-action" style="color: rgb(59, 148, 217); padding-top: 0px; padding-bottom: 0px;" href="https://twitter.com/siginthq">
+                        <img src="https://comp-soc.com/static/img/sigs/sigint-logo.png" class="img img-fluid sigint-logo" />
```

Comment:
would using `src="https://pbs.twimg.com/profile_images/788344487613493248/7iK1n0GW_400x400.jpg"` be ok?

Reply:
Yeah that should be good, in which case I can put the image rounding back on it as well to match the others